### PR TITLE
Initial code structure with setup and build scripts.

### PR DESCRIPTION
**Pull request by kaustubh-d**
Initial code structure with setup and build scripts.

-----
[View rendered README.md](https://github.com/kaustubh-d/cortx-multisite/blob/initpycode/README.md)
[View rendered s3/replication/common/README.md](https://github.com/kaustubh-d/cortx-multisite/blob/initpycode/s3/replication/common/README.md)
[View rendered s3/replication/manager/README.md](https://github.com/kaustubh-d/cortx-multisite/blob/initpycode/s3/replication/manager/README.md)
[View rendered s3/replication/replicator/README.md](https://github.com/kaustubh-d/cortx-multisite/blob/initpycode/s3/replication/replicator/README.md)

### CODE REVIEW

File: s3/replication/common/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Introduction
+Common modules for CORTX cross cluster/region replication for S3 objects.
+
+# Modules
+Following modules are defined in common package to be used by relication manager and replicator.
```
Comment (rajkumarpatel2602) [inline s3/replication/common/README.md:5]: s/relicatio/replication

File: s3/replication/common/setup.cfg [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+[metadata]
+name = s3-replication-common
+author = Seagate
+author_email = opensource@seagate.com
+description = Common modules for CORTX cross cluster/region replication for S3 objects.
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/Seagate/cortx-multisite
+project_urls =
+    Bug Tracker = https://github.com/Seagate/cortx-multisite/issues
```
Comment (rajkumarpatel2602) [inline s3/replication/common/setup.cfg:29]: Not sure, why 'Tracker' is showing as red, though not start of the line. May be indentation problem?

File: s3/replication/manager/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Replication manager for CORTX cross cluster/region replication for S3 objects.
+
+# Description
+
+This module performs following
+1. Receive "replication jobs" i.e FDMI records on S3 object create or S3 Object tag updates. FDMI records contain S3 metadata entries.
+2. Persist replication jobs in distributed Motr KVS. (Queue)
+3. Distribute jobs to subscribers.
+4. Updates S3 metadata to reflect replication status once jobs are acknowledged by subscribers.
+5. Main a list of inprogress jobs and queued jobs and return on query.
```
Comment (rajkumarpatel2602) [inline s3/replication/manager/README.md:12]: Maintain

**Review by rajkumarpatel2602 — COMMENTED** [review]
A few trivial comments are given.

File: s3/replication/manager/README.md (hunk 3 above)
Comment (kaustubh-d) [reply s3/replication/manager/README.md]: Fixed

File: s3/replication/common/setup.cfg (hunk 2 above)
Comment (kaustubh-d) [reply s3/replication/common/setup.cfg:29]: May be its just recognising it as keyword.

File: s3/replication/common/README.md (hunk 1 above)
Comment (kaustubh-d) [reply s3/replication/common/README.md]: Fixed

**Review by rajkumarpatel2602 — APPROVED** [review]
Looks great!

**Review by somnathbghule — APPROVED** [review]
Looks good to me.

**Pull request merged**